bin/oneoff/backfill_data/backfill_old_workshop_dates.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+# Script to backfill workshops, that were migrated from Workshop to Pd::Workshop,
+# with their start & stop dates, to prevent them from showing up in dashboards
+# because their states appear to be STATE_NOT_STARTED.
+
+ActiveRecord::Base.transaction do
+  cutoff_date = Date.new(2019, 2, 28)
+  total_problems = 0
+  total_updated = 0
+  total_skipped = 0
+
+  oldest_updated = Date.today
+  newest_updated = Date.new(1944, 12, 12)
+
+  Pd::Workshop.where(started_at: nil, ended_at: nil).each do |w|
+    starting_date = w.workshop_starting_date
+    ending_date = w.workshop_ending_date
+
+    if starting_date.nil? || ending_date.nil?
+      puts "Workshop #{w.id} had problem with starting/ending dates."
+      total_problems += 1
+    elsif starting_date < cutoff_date
+      puts "Update workshop #{w.id} gets new dates of #{starting_date} - #{ending_date}"
+
+      w.start_date = starting_date
+      w.ending_date = ending_date
```

Comment:
I can't find definitions for `start_date` and `ending_date` on `Pd::Workshop`.  Did you mean `started_at` and `ended_at`?

Reply:
Ah, good catch!